app/models/post.rb

```diff
@@ -7,6 +7,12 @@ class Post < ApplicationRecord
 
   scope :get_5_comments, ->(post) { post.comments.order('created_at desc').limit(5) }
 
```

Comment:
- [ ] It would be very nice if you test this method since it is a requirement that all the custome methods in models should be tested 👍